src/Plugin/Field/FieldWidget/ColorSchemeWidget.php

```diff
@@ -100,12 +101,12 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
    *   The color scheme options.
    */
   protected function getColorSchemeOptions(): array {
-    $theme_name = $this->themeManager->getActiveTheme()->getName();
-    $theme_info = $this->themeExtensionList->getExtensionInfo($theme_name);
+    $default_theme = $this->configFactory->get('system.theme')->get('default');
```

Comment:
The `theme_handler` service does the same thing by calling `->getDefault()`. But the difference seems to be minimal. It's up to you.